### PR TITLE
Registries overview page support

### PR DESCRIPTION
**Pull request by aaxelb**
## Purpose
This branch has the parts of `release/next-registries` that are required for the registries overview page. The rest of that branch (the Form Block API) can wait until next year.

## Changes
- Add wiki count to registrations (#8717)
- Add comment count to registrations (#8741)
- Redirect obviated registration views to the base guid view (#8763)
- Send new registries routes to ember-osf-web (https://github.com/CenterForOpenScience/osf.io/commit/dd02f5c4a5630ba81296dd22ee3d17392f86b5c9)
- Add `pending_embargo_termination_approval` to registration view in API (#8896)

## Ticket
https://openscience.atlassian.net/browse/EMB-429
https://openscience.atlassian.net/browse/EMB-431
https://openscience.atlassian.net/browse/EMB-432
https://openscience.atlassian.net/browse/EMB-450

### CODE REVIEW

File: api_tests/nodes/views/test_node_detail.py [hunk 1]
```diff
@@ -58,6 +59,13 @@ def project_private(self, user):
             is_public=False,
             creator=user)
 
+    @pytest.fixture()
+    def wiki_private(self, user, project_private):
```
Comment (sloria) [inline api_tests/nodes/views/test_node_detail.py]: Is this fixture used anywhere?

Comment (aaxelb) [reply api_tests/nodes/views/test_node_detail.py]: Doesn't look like it! Removed.

File: api/registrations/serializers.py [hunk 2]
```diff
@@ -117,7 +117,7 @@ class RegistrationSerializer(NodeSerializer):
         related_view_kwargs={'node_id': '<_id>'},
         related_meta={
             'unread': 'get_unread_comments_count',
-            'total': 'get_total_comments_count',  # total count of top_level, undeleted comments on the registration (node)
+            'count': 'get_total_comments_count',
```
Comment (sloria) [inline api/registrations/serializers.py]: Isn't this a breaking change? Is this a documented property?

Comment (aaxelb) [reply api/registrations/serializers.py]: Nope, `total` was added earlier in this PR. [Previously it had just `unread`](https://github.com/CenterForOpenScience/osf.io/pull/8876/files#diff-c4e5a8ca4d9df4503422cdf682b3235bL114)

**Comment by sloria** [comment]
Just one question, otherwise looks good to go.

File: api/registrations/serializers.py (hunk 2 above)
Comment (aaxelb) [reply api/registrations/serializers.py]: (`related_counts` causes special handling of `related_meta['unread']` and `related_meta['count']`, so `total` was *always* included. Switching to `count` seemed cleaner than adding yet another special case)

Comment (sloria) [reply api/registrations/serializers.py]: Ah, makes sense. Thanks for clarifying.

**Pull request merged**